Rock.code-workspace

```diff
@@ -72,7 +72,7 @@
 		"html.format.wrapAttributes": "preserve-aligned",
 		"html.format.wrapLineLength": 0,
 		"editor.codeActionsOnSave": {
-			"source.fixAll.eslint": true
+			"source.fixAll.eslint": "explicit"
```

Comment:
What does this change do? Do we need it? Does it make sense for it to be a part of the HubSpot changes?